mods/boats/init.lua

```diff
@@ -180,15 +171,13 @@ function boat.on_step(self, dtime)
 		self.object:set_pos(self.object:get_pos())
 		return
 	end
-	local s = get_sign(self.v)
-	self.v = self.v - dtime * 0.6 * s
-	if s ~= get_sign(self.v) then
-		self.object:set_velocity({x = 0, y = 0, z = 0})
+	-- We need to preserve velocity sign to properly apply drag force while moving backward
+	local drag = dtime * math.sign(self.v) * (0.01 + 0.08 * self.v * self.v)
+	-- If drag is larger than velocity, then stop horizontal movement.
```

Comment:
Single line comment so no full stop needed (see other comments)

Reply:
Could you please explain what I need to change in this comment?
I think the comment is correct (we need to set velocity to 0, to avoid negative values caused by drag).

Reply:
Sorry if i was unclear, all i request is removing the '.' at the end, single-line comments do not require them, see the other single-line comments in the file.

Reply:
In English the '.' character is sometimes called a 'full stop', maybe this caused confusion =D

Reply:
Actually nevermind this, it is only a tiny change, i will merge this now.